forms-html/Shirin-MohammedKhalil/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Shirin-Mohammed</title>
+  </head>
+  <body>
+    <form action="#" method="POST">
+      <!-- Email-->
+      <label for="email">Enter Your E-mail </label><br />
+      <input
+        type="email"
+        id="email"
+        name="email"
+        required
+        placeholder="Shirin-Mohammed@gmail.com"
+      />
```

Comment:
Great job on using the for attribute.
This is the best practice. Now the user can click on the label and its corresponding field get focused!